### ISSUE TITLE
Outgoing requests and error logging to stdout

### ISSUE THREAD

**Issue by Coperh** [open-zaak/open-zaak#1506]
Ref DH 528

Request from Den Haag is for v1.11 to also see the outgoing requests (successes and errors/tracebacks) on stdout

**Description**

The usecase is that the logs from Open Zaak et al are collected centrally via grafana so more visibility via stdout will ensure an easier time when it comes to debugging connection issues.

The suggestion is either to expand what is written to stdout by default by Open Zaak, or otherwise to introduce an environment variable to switch between minimal stdout logging and more expansive logging.

By default it would be expected that our django-log-outgoing-requests to be off, to avoid impact on the performance of Open Zaak an admin has to activate it manually via the admin.

**Comment by annashamray** [comment]
@alextreme 
I think it's already done. I'll describe what is already implemented, please comment if something is missing

**How it works now:**
We have environment variable `LOG_REQUESTS` with default value = "True". If this var is True, then we we use two handlers to log outgoing requests:
* handler to write into stdout, which works all the time
* handler to write into DB, which checks the config in the admin to know, should we write in DB

To summarize if you don't specify `LOG_REQUESTS=False` in your env your outgoing requests would be always logged in the stdout

Is this configuration what you want? Or do you want something else?

**Comment by alextreme** [comment]
@annashamray from your comment I'm not sure if the following is the case:

> By default it would be expected that our django-log-outgoing-requests to be off, to avoid impact on the performance of Open Zaak an admin has to activate it manually via the admin.

But besides that this issue has then been completed, I suspect django-lor was then added via a different issue :+1:

**Comment by annashamray** [comment]
Yes, it was Joeri's US to add django-lor. 
I'll change the default value
